declarations/Speechify.json

```
@@ -11,8 +11,8 @@
       "fetch": "https://speechify.com/privacy/",
       "select": [
         {
-          "startBefore": "#cookiepolicy",
-          "endBefore": "[id=\"6\"]"
+          "startBefore": "#cookie-policy",
+          "endBefore": "#contact-us"
```

This contact us section is a part of the Trackers Policy
```suggestion
          "endAfter": "#contact-us"
```

---

You're right, but your suggestion doesn't allow to select the paragraph after the title "Contact Us".

---

What do you think of
```suggestion
          "endBefore": ".footertopwave"
```

---

> You're right, but your suggestion doesn't allow to select the paragraph after the title "Contact Us".

Yes, sorry, I made a proposal too quickly.

---

It seems to me that this is more robust:
```suggestion
          "endAfter": "main"
```

---

> Yes, sorry, I made a proposal too quickly.

Don't worry, and thank you for pointing out this error ;)